src/trading/utils/slippage.ts

```diff
@@ -0,0 +1,51 @@
+const SCALE = 10n ** 6n // 6 decimal places of precision. Used to avoid depending on Big Decimal libraries
+
+/**
+ * Get the slippage percentage for a given absolute slippage.
+ *
+ * Returns a percentage as a portion of 1 with 6 decimals of precision.
+ *
+ * @param params
+ * @returns
+ */
+export function getSlippagePercent(params: {
+  isSell: boolean
+  sellAmountBeforeNetworkCosts: bigint
+  sellAmountAfterNetworkCosts: bigint
+  slippage: bigint
+}): number {
+  const { sellAmountBeforeNetworkCosts, sellAmountAfterNetworkCosts, slippage, isSell } = params
+
+  const sellAmount = isSell ? sellAmountAfterNetworkCosts : sellAmountBeforeNetworkCosts
+
+  // Check that the sellAmount after accounting for the fee is not negative
+  if (sellAmount <= 0n) {
+    throw new Error('sellAmount must be greater than 0: ' + sellAmount)
+  }
```

Comment:
Same here, sellAmount must be > 0

Reply:
But this is asserting the `sellAmount is positive, right?

Reply:
https://github.com/cowprotocol/cow-sdk/pull/316